csrc/scheduler/resize.cpp

```diff
@@ -254,12 +326,12 @@ void ResizeScheduler::schedule(Fusion* fusion, const HeuristicParams* params) {
 
   scheduler_utils::clearMemorySpace(fusion);
 
-  auto ref_tv = getReferenceTensor(fusion);
-  NVF_ERROR(ref_tv != nullptr);
-
   scheduler_utils::cacheInputs(fusion, true);
   scheduler_utils::cacheAndForkOutputs(fusion, true);
 
+  auto ref_tv = getReferenceTensor(fusion);
```

Comment:
This change is not strictly required, but just to pick a reference from fusion outputs. `cacheAndForkOutputs` may change output tensors.